src/modules/stream/internal.js

```diff
@@ -73,6 +75,36 @@ async function handleYoutubeStream(streamInfo, res) {
     }
 }
 
+function transformHLSMediaPlaylist(streamInfo, hlsPlaylist) {
```

Comment:
rename the function to something shorter and follow camelcase even for abbreviations, as it's more readable this way: `transformHlsPlaylist`